server/src/controllers/user.controller.ts

```diff
@@ -99,4 +100,16 @@ export class UserController {
   async getProfileImage(@Res() res: Response, @Next() next: NextFunction, @Param() { id }: UUIDParamDto) {
     await sendFile(res, next, () => this.service.getProfileImage(id), this.logger);
   }
+
+  @Put('me/license')
+  @Authenticated()
+  async setUserLicense(@Auth() auth: AuthDto, @Body() license: LicenseKeyDto): Promise<LicenseResponseDto> {
+    return this.service.setLicense(auth, license);
+  }
+
+  @Delete('me/license')
+  @Authenticated()
+  async deleteUserLicense(@Auth() auth: AuthDto): Promise<void> {
+    await this.service.deleteLicense(auth);
+  }
```

Comment:
What do you think about putting all the me variants at the top of the controller? Just like because I hate wildcard routes somehow getting added later and causing problems.